vmdb/spec/models/ems_refresh/refreshers/foreman_refresher_spec.rb

```diff
@@ -134,6 +131,29 @@ def assert_configured_system
     expect(system.configuration_profile).to   eq(child)
   end
 
+  it "loads data with locations" do
```

Comment:
I'd rather not have a second refresh test to test the locations.  Can we just have one test with everything?  I've tried this in the past but EMS Refresh specs are expensive, since they are really full stack integration tests.